make_sysd_rules.py

```diff
@@ -171,7 +171,9 @@ def parse_syscalls_list(path):
     "sysdeps/nacl/wait4.c",
     "sysdeps/nacl/sigprocmask.c",
     "sysdeps/nacl/lstat.c",
-    "sysdeps/nacl/gethostname.c"
+    "sysdeps/nacl/gethostname.c",
```

Comment:
gethostname again?

Reply:
No, I just added a comma in the en since it is a list and a comma is now required:

Old version: [..., "sysdeps/nacl/gethostname.c"]

New version: [ … ,"sysdeps/nacl/gethostname.c","sysdeps/nacl/pread.c", "sysdeps/nacl/pwrite.c"]